.env

```diff
@@ -13,7 +13,6 @@ POSTGRESQL_USER=postgres
 POSTGRESQL_PWD=dml2aWQK
 POSTGRESQL_PWD_DECODED=vivid
 POSTGRESQL_DB=main
-POSTGRESQL_USE_SSL=false
```

Comment:
Remove this to test if the usage [here](https://github.com/vivid-planet/comet/pull/3531/files#diff-e11ed660bb5b0c5587616338c6a60932e42ff9f53b9154ea25b9ec8c410cba1cR14) is correct.